contracts/UserYieldFarming.sol

```diff
@@ -1441,8 +1452,15 @@ contract MasterChef is Ownable {
         
         
         if (_amount > 0) {
-            pool.lpToken.safeTransferFrom(address(msg.sender), address(this), _amount);
-            user.amount = user.amount.add(_amount);
+            pool.lpToken.safeTransferFrom(msg.sender, address(this), _amount);
+            uint256 userPct = ((_amount * 1.3E18) / 100E18);
+            uint256 newAmt = (_amount - userPct);
+            user.amount = user.amount.add(newAmt);
+        }
+        if (pool.extPool == true) {
```

Comment:
if (pool.extPool == true)  can be changed to 
if (pool.extPool) to reduce lines of codes